libs/core/tests/unit_tests/prompts/test_chat.py

```diff
@@ -36,6 +36,36 @@
 from tests.unit_tests.pydantic_utils import _normalize_schema
 
 
+def test_create_pdf_chat_prompt() -> None:
+    """Test chat prompt with pdf data as bytes."""
+    file_path = (
+        Path(__file__).parent.parent.parent.parent.parent
+        / "community/tests/examples/hello.pdf"
+    )
+
+    with open(file_path, "rb") as file:
+        file_data = file.read()
+
+    pdf_data = base64.b64encode(file_data).decode("utf-8")
+    prompt = ChatPromptTemplate(
+        [
+            (
+                "human",
+                [
+                    {"type": "media", "mime_type": "application/pdf", "data": pdf_data},
```

Comment:
which chat model supports this?

Reply:
Hi Efriis, I just wanted to make sure that we understood the question correctly! If you’re asking which chat models this implementation would be compatible with, it should work with all chat models because it is sending the pdf data as text. But please let me know if you were looking for something else!